test/integration/note_creation_test.rb

```diff
@@ -0,0 +1,34 @@
+require 'test_helper'
+
+class NoteCreationTest < ActionDispatch::IntegrationTest
+  before do
+    @user = create :user
+    log_in_as @user
+    @patient = create :patient
+    @pregnancy = create :pregnancy, patient: @patient
+    visit edit_pregnancy_path(@pregnancy)
+  end
+ 
+  describe 'add patient pregnancy notes' do
+    it 'should display a case notes form and current notes' do
+        click_link 'Notes'
+        
+        visit current_path 
```

Comment:
IIRC the controller action currently redirects to the view -- is it possible to take out the `visit current_path` here?